Apps/WebClient/src/ClientApp/src/components/modal/ProtectiveWordComponent.vue

```diff
@@ -34,15 +36,19 @@ export default class ProtectiveWordComponent extends Vue {
     private logger!: ILogger;
 
     private get isVisible(): boolean {
-        return this.isProtected && !this.isLoading && !this.isDismissed;
+        return (
+            this.medicationsAreProtected(this.user.hdid) &&
+            !this.isLoading &&
+            !this.isDismissed
+        );
     }
 
     private set isVisible(visible: boolean) {
         this.isDismissed = !visible;
     }
 
     private get error(): boolean {
-        return this.protectedWordAttempts > 1;
+        return this.protectiveWordAttempts(this.user.hdid) > 1;
```

Comment:
I wonder if this makes sense for dependents

Reply:
Good question.  I don't think we know yet.